### ISSUE TITLE
Pronto raises cop errors

### ISSUE THREAD

**Issue by CoderMiguel** [prontolabs/pronto-rubocop#88]
I am experiencing rubocop warning while running pronto that are not reproducible when running rubocop directly.

The only issue or mention of the issue that I am seeing is issue: https://github.com/prontolabs/pronto-rubocop/issues/76

I have updated `rubocop` and `pronto` / `pronto-rubocop`
- rubocop (1.59.0)
- pronto (0.11.2)
- pronto-rubocop (0.11.5)
- ruby 3.1.3 (other gems are limiting me from using 3.1.4 atm)

any thoughts on how to address this or something I can do to help?

a tallied list of the cops that are throwing these 78 errors:
{"Lint/MissingCopEnableDirective"=>11,
 "Lint/RedundantCopEnableDirective"=>12,
 "Naming/AsciiIdentifiers"=>1,
 "Style/IfUnlessModifier"=>28,
 "Metrics/MethodLength"=>4,
 "Metrics/AbcSize"=>4,
 "Style/MultilineIfModifier"=>1,
 "Layout/EmptyLineAfterGuardClause"=>1,
 "Lint/UselessRescue"=>1,
 "Style/RescueStandardError"=>2,
 "Layout/CommentIndentation"=>1,
 "Lint/MixedRegexpCaptureTypes"=>2,
 "Layout/MultilineMethodCallIndentation"=>8,
 "Metrics/ParameterLists"=>2}
 
 of the files that are effected 2 of them are trigger 39 of these instances and both of them yield no errors when running `rubocop -d path_to/file.rb`

```console
For /path_to/ProjectName: configuration from /path_to/.rubocop.yml
Default configuration from /path_to/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/rubocop-1.57.2/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /path_to/file.rb
.

1 file inspected, no offenses detected
Finished in 0.14713699975982308 seconds
```

**Comment by ashkulz** [comment]
@CoderMiguel do you have a reproduction for this?

**Comment by CoderMiguel** [comment]
@ashkulz no, it's been over a year since I posted this issue.  I don't even remember the work around we used to resolve it.

Closing
